.circleci/set-up-globals.sh

```diff
@@ -0,0 +1,129 @@
+#!/bin/bash
+
+set -ex
+
+#==================================================
+# The section would be transferable to a DOCKERFILE
+#==================================================
+
+# Update current apt packages
+apt-get update
+
+#=========================================================================
+# Commands below this line would not be transferable to a docker container
+#=========================================================================
+
+# Enable Composer parallel downloads
+composer global require -n "hirak/prestissimo:^0.3"
+
+# Install Terminus
+if [ ! -d $HOME/terminus ]
+then
+	# Clone terminus if it doesn't exist
+	echo -e "Installing Terminus...\n"
+	git clone --branch master git://github.com/pantheon-systems/terminus.git $HOME/terminus
```

Comment:
Previously, this was installed via Composer (but not via `composer global require` - see https://pantheon.io/blog/fixing-composer-global-command). However, @ataylorme wanted to cache and update terminus rather than re-install every time, so we are experimenting with different installation methods.

In other projects, I install Terminus [using composer require like this](https://github.com/pantheon-systems/example-drops-8-composer/blob/882279011083b1d39d7d8f91a413ae554d5a98e8/circle.yml#L47). This installation mechanism should be updatable via a `cd` + `composer update`.